CONTRIBUTING.md

```diff
@@ -1,27 +1,33 @@
 # Contribution Guidelines
-Unfortunately, not every library, tool or framework can be considered for inclusion. The aim of Awesome PHP is to be a concise list of noteworthy and interesting software written in modern PHP. Therefore, suggested software should:
+The aim of Awesome PHP is to be a concise list of the best packages, frameworks, tools and software the PHP ecosystem has to offer. Unfortunately, this means not everything can or should be included. Therefore, suggested software should ideally be one of the following:
```

Comment:
- ~~The aim of Awesome PHP is to~~ Awesome PHP aims to 
- ~~tools and~~ tools, and
- ~~suggested software~~ the suggested software